src/ga.js

```diff
@@ -179,7 +187,7 @@ function sendBidRequestToGa(bid) {
   if (bid && bid.bidderCode) {
     _analyticsQueue.push(function () {
       _eventCount++;
-      window[_gaGlobal]('send', 'event', _category, 'Requests', bid.bidderCode, 1, _disibleInteraction);
+      window[_gaGlobal]( trackerSend, 'event', _category, 'Requests', bid.bidderCode, 1, _disibleInteraction);
```

Comment:
@protonate not sure how that happens. Shouldn't `enableAnalytics` always get called before this function, therefore setting `trackerSend`?